### PR TITLE
add tests for resource_pipeline

### PR DESCRIPTION
**Pull request by Isakgicu**
## WHAT

Add tests for resource_pipeline.

## WHY

Pipeline resource should also have tests.

### CODE REVIEW

File: spinnaker/resource_pipeline_test.go [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+func TestAccResourceSourceSpinnakerPipeline_basic(t *testing.T) {
+	resourceName := acctest.RandomWithPrefix("tf-acc-test")
+	application := acctest.RandomWithPrefix("tf-acc-test")
+	pipeline := `{
+  		"keepWaitingPipelines": false,
+  		"limitConcurrent": true,
+  		"notifications": [ ],
+  		"stages": [ ],
+  		"triggers": [ ]
+		}`
```
Comment (KeisukeYamashita) [inline spinnaker/resource_pipeline_test.go:22]: Can you remove the extra space, please 🙏 

```suggestion
	pipeline := `{
  		"keepWaitingPipelines": false,
  		"limitConcurrent": true,
  		"notifications": [],
  		"stages": [],
  		"triggers": []
		}`
```

**Review by KeisukeYamashita — DISMISSED** [review]
LGTM, thank you very much 🎸

**Review by KeisukeYamashita — COMMENTED** [review]
tiny nits except LGTM

File: spinnaker/resource_pipeline_test.go (hunk 1 above)
Comment (Isakgicu) [reply spinnaker/resource_pipeline_test.go]: Hi, updated last commit. 
Awaiting Your approval for merge as I'm looking to make another pull request with terraform import feature for pipelines, and tests for this feature as well.

Comment (KeisukeYamashita) [reply spinnaker/resource_pipeline_test.go]: Thank you for your update 👍

**Review by KeisukeYamashita — APPROVED** [review]
LGTM 👍  Thank you!

**Pull request merged**